java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/DefaultDockerfileGenerator.java

```diff
@@ -46,17 +46,16 @@
 
   @Override
   public String generateDockerfile(Path artifactToDeploy) {
-    String fileType = FileUtil.getFileExtension(artifactToDeploy);
+    String fileType = Files.getFileExtension(artifactToDeploy.toString());
     String baseImage;
     String appDest;
 
-    // TODO both runtimes should suppport adding the application to the same destination
     if (fileType.equals("jar")) {
       baseImage = jarRuntime;
       appDest = "app.jar";
     } else if (fileType.equals("war")) {
       baseImage = serverRuntime;
-      appDest = "/app";
+      appDest = System.getenv("JETTY_BASE") + "/webapps/root.war";
```

Comment:
I think this just has to be `$JETTY_BASE/webapps/root.war`. It will be resolved during docker build.